Jenkinsfile

```diff
@@ -397,10 +390,10 @@ stage('Unit Test') {
           }
         }
       } else {
-        Utils.markStageSkippedForConditional('python3: GPU')
+        Utils.markStageSkippedForConditional('unittest: GPU')
       }
     },
-    'python3: CPU': {
```

Comment:
i think right now, failing unit tests wouldn't cancel integration tests. are we concerned with overburdening CI with PRs that fail unit tests? I wonder if we should somehow cancel integration test builds if unit tests fail. we could also just try merging and see if it's a problem, too.

Reply:
The potential time savings of this are pretty significant so I think it's worth it even if it demands more capacity from CI. I could try to pull some numbers of # of jobs vs # of jobs with failing unit tests vs # of jobs with failing integration tests to back this up

Reply:
Here are some classifications over the last several failing PR jobs, seems like the unit test failures aren't super frequent compared to others so this PR is probably fine in terms of demand increase 

```
build	262
lint	91
integration	46
infra	22
unit tests	19
unclassified	3
```

Reply:
ok, main concern here is about the frontend tests i think. those do take over an hour to run, so that might be a big load increase if they don't get cancelled.

Reply:
sorry should have included this, but this is how I did the accounting:

<details>

```
WHEN name like '%Build and run C++ tests' THEN 'build'
WHEN name like '%Run cmake build' THEN 'build'
WHEN name like '%Run microTVM tests' THEN 'build'
WHEN name like '%integration tests' THEN 'integration'
WHEN name like '%frontend tests' THEN 'integration'
WHEN name like '%unit tests' THEN 'unit tests'
WHEN name like '%Sphinx warnings in docs' THEN 'lint'
WHEN name like '%Run lint' THEN 'lint'
WHEN name like '%executor node info' THEN 'infra'
WHEN name like '%Check out from version control' THEN 'infra'
WHEN name like '%JUnit-formatted test results' THEN 'infra'
WHEN name like '%Docker image names' THEN 'infra'
WHEN name like '%files previously stashed' THEN 'infra'
WHEN name like '%Rust build and test' THEN 'build'
```

</details>

Isn't the concern the case where unit tests that would have failed and then caused frontend (integration) tests to not run, so the number to look at is unit test failures (since with this PR those would no longer be gating frontend failures)? Frontend failures would mean that the build got all the way there anyways so the capacity requirements of those would be the same

Reply:
ok, given the data i support trying this change! we may need to evaluate its impact as folks start using it day-to-day.